docker/latest/consumers/consumers.properties

```diff
@@ -12,3 +12,13 @@ consumer.workload.consumers.per.subscription=1
 
 schema.cache.refresh.after.write.minutes=1
 schema.repository.serverUrl=http://schema-registry:8081
+
+#kafka.authorizaiton.enabled=true
```

Comment:
Until the keystore and truststore are added to the images in my opinion, it is unnecessary to leave these properties commented out.